client/lib/post-normalizer/rule-create-better-excerpt-refresh.js

```diff
@@ -43,6 +43,10 @@ export default function createBetterExcerpt( post ) {
 		return post;
 	}
 
+	// Create standard excerpt for Discover
+	post.excerpt_no_html = formatExcerpt( post.excerpt );
```

Comment:
The excerpt sometimes comes wrapped in a `<p>`, which is all I'm wanting to strip here.

If we don't remove this wrapping paragraph, we'd have to display the excerpt using `dangerouslySetInnerHTML`, which feels wrong.